_jugs/OttawaJUG.md

```diff
@@ -0,0 +1,7 @@
+---
+name:     "Ottawa Java User Group"
```

Comment:
It is interesting to add the `founded_date` to know how old each JUG is 😉 
(but of course, it is not mandatory... 😅)

Reply:
Sure, no problem! Let's merge your PR and thanks for your contribution! ;-)